cores/nRF5/Adafruit_TinyUSB_Core/tinyusb/src/class/cdc/cdc_device.h

```diff
@@ -57,10 +57,10 @@ void        tud_cdc_n_get_line_coding (uint8_t itf, cdc_line_coding_t* coding);
 void        tud_cdc_n_set_wanted_char (uint8_t itf, char wanted);
 
 uint32_t    tud_cdc_n_available       (uint8_t itf);
-signed char tud_cdc_n_read_char       (uint8_t itf);
+int         tud_cdc_n_read_char       (uint8_t itf);

```

Comment:
would you mind changing all `int` to `int32_t`, since this is part of tinyusb stack, I would prefer to stick with known size return value. Since the stack isn't limited to 32-bit mcu.

Reply:
I've added a commit to do int -> int32_t